### DISCUSSION TITLE
希望能加入收藏夹修复组件

### DISCUSSION THREAD

**Discussion by ChaosDeus** [the1812/Bilibili-Evolved#5085] [功能建议 / Ideas]
曾经用过一个名为“哔哩哔哩(B站|Bilibili)收藏夹Fix”的脚本，现在由于b站框架重构已经失效了，项目地址https://github.com/MotooriKashin/Bilibili-Old，油猴地址https://greasyfork.org/zh-CN/scripts/383143
在整理收藏夹时发现，对于新版个人主页，目前没有比较好的组件进行旧版的重现，希望至少能把收藏夹失效视频进行修复。

**Comment by the1812** [comment]
![Snipaste_2024-12-28_11-38-58](https://github.com/user-attachments/assets/99e7d510-e57b-49b6-a31b-5aa026094a1e)

**Comment by crnkv** [comment]
你可能不知道：[哔哩哔哩(B站|Bilibili)收藏夹Fix (cerenkov修改版)](https://greasyfork.org/zh-CN/scripts/489224)

**Reply by rivzzzzz** [reply]
又寄了（悲

**Reply by crnkv** [reply]
> 又寄了（悲

由于现在B站官方APP能显示失效视频的标题和封面了，所以我的脚本可能之后有空闲时间的话会修，但优先级不高
